readme.md

```diff
@@ -88,4 +97,42 @@ The metric that would actually measure the business outcome is transmission rate
 
 ### Icebox
 1. “Initiative2.epic3.story2” 
-2. “Initiative2.epic4” 
+2. “Initiative2.epic3.story3” 
+3. “Initiative2.epic4” 
+
+## <a name="#instructions">Instructions
+
+### Step 1. Clone the repository
+```
+git clone git@github.com:qinlang0628/2020-msia423-Qin-Covid19.git
+cd 2020-msia423-Qin-Covid19
+git checkout development
+```
+### Step 2. Build the docker
```

Comment:
In terms of wording, image should be added after docker.

Reply:
Editted.